### ISSUE TITLE
in version 2.1.7 drag & drop is broken

### ISSUE THREAD

**Issue by smilingkite** [phphe/vue-draggable-nested-tree#39]
I had two installs - one with 2.1.6 , one with 2.1.7 - in both cases upgrading to 2.1.7 means that  the drag and drop no longer works. One of them is a pretty clean vue-cli-3 install. 

Downgrading to 2.1.6 fixes it.

**Comment by NeoMarine** [comment]
This issue was introduced from a fix for [this issue](https://github.com/phphe/vue-draggable-nested-tree/issues/37).

To use 2.1.7, you'll have to set cross-tree="cross-tree" (or :cross-tree="true")

**Comment by smilingkite** [comment]
This works if I set cross-tree to true, but not when I set it to false.

**Comment by NeoMarine** [comment]
Yes, it is currently broken and requires cross-tree to be true. Waiting on a fix @phphe :)

**Comment by smilingkite** [comment]
My fix for the dragging problem also fixes this.

https://github.com/phphe/vue-draggable-nested-tree/issues/42
